jib-core/src/main/java/com/google/cloud/tools/jib/image/json/ContainerConfigurationTemplate.java

```diff
@@ -127,12 +130,14 @@ public void setContainerCmd(List<String> cmd) {
     config.Cmd = cmd;
   }
 
-  public void setContainerExposedPorts(List<String> exposedPorts) {
+  public void setContainerExposedPorts(List<PortsWithProtocol> exposedPorts) {
```

Comment:
This function should probably still take a `String`, and the list of strings should be built somewhere else, and same for the getter.

Reply:
I'll take care of that when I get rid of the TODO